internal/policygen/load.go

```diff
@@ -94,33 +100,35 @@ func loadConfig(path string) (*config, error) {
 	return c, nil
 }
 
-// loadResources loads Terraform state resources from the given path. If the path is a single
-// file, it loads resouces from it. If the path is a directory, it walks the directory
-// recursively and loads resources from each .tfstate file.
+// loadResources loads Terraform state resources from the given path.
+// - If the path is a single local file, it loads resouces from it.
+// - If the path is a local directory, it walks the directory recursively and loads resources from each .tfstate file.
+// - If the path is a GCS bucket (indicated by 'gs://' prefix), it walks the bucket recursively and loads resources from each .tfstate file.
+//   It currently ignores the file/dir path inside the bucket and considers all .tfstate file in the bucket.
```

Comment:
Is it difficult to consider the file or dir in the bucket?

Reply:
It's not difficult but just will make the code a bit messy and we currently don't need that. We can add that when there is a FR for that.

Reply:
sg